docs/index.md

```diff
@@ -38,10 +36,10 @@ Both NPM and Yarn have a lockfile, which ensures that dependencies only change w
 
 References:
 
-- npm: https://docs.npmjs.com/files/package-lock.json
-- yarn: https://yarnpkg.com/lang/en/docs/yarn-lock/
+- npm: [https://docs.npmjs.com/files/package-lock.json](https://docs.npmjs.com/files/package-lock.json)
```

Comment:
nit: instead of repeating the text twice, use autolink: https://daringfireball.net/projects/markdown/syntax#autolink

`<https://docs.npmjs.com/files/package-lock.json>`